core/src/tx.rs

```diff
@@ -91,30 +91,27 @@ impl TransactionValidator {
     /// Fails if validation of any transaction fails
     pub fn validate_every(
         &self,
-        txs: &[VersionedAcceptedTransaction],
+        txs: impl IntoIterator<Item = VersionedAcceptedTransaction>,
```

Comment:
Yes, but now we need to capture `txs` by value, because every transaction will be moved into runtime validator for validation (which will be stored in the `State`)

Reply:
Ok, we can do a research for that. I kind off agree with you. But right now `tx` has do be converted to another state of `Transaction`. That makes things complicated.
The easiest way for now: add a TODO and create an issue

Reply:
Created #2742